items/resources.py

```diff
@@ -116,6 +116,16 @@ class Meta:
     paginator_class = Paginator
     limit = 10
 
+  def hydrate_users(self, bundle):
+    body = json.loads(bundle.request.body)
+    bundle.obj.users.clear()
+    for user in body['users']:
+      user = User.objects.get(pk=user['id'])
+      bundle.obj.users.add(user)
+    if (bundle.obj.creator not in bundle.obj.users.all()):
+      bundle.obj.users.add(bundle.obj.creator)
```

Comment:
maybe it's better to change `GroupAuth` then? to enable doing things for creator personally, no?